job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PlotPluginContext.groovy

```diff
@@ -0,0 +1,19 @@
+package javaposse.jobdsl.dsl.helpers.publisher
+
+import javaposse.jobdsl.dsl.ContextHelper
+import javaposse.jobdsl.dsl.Context
+
+class PlotPluginContext implements Context {
```

Comment:
Rename this class to PlotContext, the other contexts also do not contain "plugin" in their name.